.travis.yml

```diff
@@ -3,7 +3,7 @@ language: java
 jdk:
   - openjdk12
   - openjdk11
-  - oraclejdk8
+  - openjdk8
```

Comment:
Hi @hazendaz In this case, we should change `after_success.sh` for deploying snapshot version.